test/parser-recovery.js

```diff
@@ -87,14 +91,17 @@ describe('Invoke parser with incomplete expression', function() {
                         "value": "Account",
                         "type": "name",
                         "position": 7,
-                        "predicate": [
+                        "stages": [
                             {
+                                "expr": {
                                 "type": "error",
                                 "error": {
                                     "code": "S0207",
                                     "position": 8,
                                     "token": "(end)"
-                                }
+                                }                                },
```

Comment:
Some strange formatting going on here; surprised the linter didn't pick this up.

Reply:
fixed